src/dummy/CommandTask.cpp

```diff
@@ -0,0 +1,56 @@
+#include "CommandTask.hpp"
+#include "Arduino.h"
+#include "ArduinoJson.h"
+
+void CommandTask::activity()
```

Comment:
Confused about what this task is accomplishing? Is it just to show you've got connection on multiple ports going?

Reply:
nothing interesting, but this is all part of the dummy target so IMO fine to include

Reply:
its just a TCP echo server